package.json

```diff
@@ -367,6 +367,7 @@
     ]
   },
   "resolutions": {
+    "@storybook/react-docgen-typescript-plugin": "1.0.6--canary.9.0c3f3b7.0",
```

Comment:
This was a recommended workaround from https://github.com/storybookjs/storybook/issues/21642#issuecomment-1472903236

I've updated it to the latest canary version of v1.0.6.